docker/release/nanoserver/Dockerfile

```diff
@@ -0,0 +1,59 @@
+# escape=`
```

Comment:
Yeah the inline makes using it simpler, it's just more duplicate code. Granted I ended up having to copy the bootstrap script into each folder anyway... I'm fine with changing it to be inline.